clj-http-lite/test/martian/clj_http_lite_test.clj

```diff
@@ -36,7 +36,19 @@
              (:body response))))))
 
 (deftest openapi-bootstrap-test
-  (let [m (martian-http/bootstrap-openapi openapi-url)]
+  (let [m (martian-http/bootstrap-openapi openapi-url)
+        mt (martian-http/bootstrap-openapi openapi-test-url)
```

Comment:
I would inline mt, mt1, mt2 below as they are only referenced once

Reply:
Okay.  I will do so.

The reason they were in the let was so that we could add additional assertions later as needed using the same martian instance.

Reply:
Also, the code with the let is consistent with the test code in `cljs_http_promise_test.cljs` that uses the `prom/let` construct.